packages/components/src/components/wizard/wizard.tsx

```diff
@@ -1,42 +1,57 @@
-import PropTypes from 'prop-types';
+import PropTypes, { object } from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
-import Step from './wizard-step.jsx';
+import Step from './wizard-step';
+
+type TWizard = {
+    className?: string;
+    initial_step: number;
+    onStepChange: (prop: { [key: string]: number }) => void;
+    nav: React.ReactNode;
+    selected_step_ref: () => any | React.MutableRefObject<HTMLElement>;
```

Comment:
Don’t understand exactly the sens of this statement if we use 'any', why we should add some other types?, I believe in this case all passed values will be determined like 'any'
correct me please

Reply:
Will change this